src/checker/http_checker.rs

```diff
@@ -63,7 +67,11 @@ fn dns_error(err: &reqwest::Error) -> Option<String> {
 
 impl HttpChecker {
     pub fn new() -> Self {
+        let mut default_headers = HeaderMap::new();
+        default_headers.insert("User-Agent", UPTIME_USER_AGENT.to_string().parse().unwrap());
```

Comment:
Shouldn't we be using the agent from the config?

Reply:
we don't have it in the config yet do we?

Reply:
We can slap it in there if we want

Reply:
Oh right, yeah we can do that whenever then. It's more important for when people are able to configure their own uptime monitors